modules/apps/commerce/commerce-frontend-js/.npmbundlerrc

```diff
@@ -0,0 +1,9 @@
+{
+	"config": {
+		"imports": {
+			"frontend-taglib-clay": {
```

Comment:
I think this shouldn't be necessary because [it's already here](https://github.com/liferay-frontend/liferay-portal/blob/1056f140ae8e5d08833af1899f1b37829c5610a4/modules/npmscripts.config.js#L184)

Reply:
@izaera 

Unfortunately I get dependency resolution errors if I don't include that, as we agreed with @markocikos that the additional work requires more changes to have everything look neater than it currently looks. So, since I'm still importing not from a `frontend-taglib-clay` proper interface but rather drilling towards the resources I need from `commerce-frontend-js` not to break anything there, I think I still need it. :(